keras_cv/models/clip/clip_image_encoder.py

```diff
@@ -0,0 +1,58 @@
+# encode images
+from collections import OrderedDict
+from typing import Tuple
+from typing import Union
+
+import numpy as np
+import tensorflow as tf
+import torch
+import torch.nn.functional as F
+from tensorflow.keras import layers
+from torch import nn
+
+from deepvision.layers.clip_patching_and_embedding import CLIPPatchingAndEmbedding
+from deepvision.layers.residual_transformer_encoder import ResidualTransformerEncoder
+from deepvision.utils.utils import parse_model_inputs
+
+
+class __CLIPImageEncoderTF(tf.keras.Model):
```

Comment:
Remove the underscores in class names

Reply:
Yep, planned, they're used only internally in the other repo so they're underscored and another API is exposed. This'll be removed when the PR is ready for review